rpm/ofono.spec

```diff
@@ -61,6 +61,13 @@ Requires:  %{name} = %{version}-%{release}
 %description doc
 Man pages for %{name}.
 
+%package -n ofono-configs-dummy
```

Comment:
I wonder if it would make sense rather than adding this package change the "Requires: ofono-configs" to weaker dependency "Recommends: ofono-configs" as the effect here would be exactly the same.